### PR TITLE
Add basic documentation about the various types of logs that AMO has

### PR DESCRIPTION
**Pull request by diox**
Fixes: mozilla/addons#15047

### CODE REVIEW

File: docs/topics/logs.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Logs
+
+(logs)=
+
+We have various types of logs with different purposes in AMO. The following
+tables summarize their characteristics:
+
+
+| | `ActivityLog` / `LogEntry` |
```
Comment (KevinMind) [inline docs/topics/logs.md:9]: Did you consider using a markdown table or at least a multi column table? Seems the rows are repeated for each type and wondered if it's more readable as a single table? I downloaded the built docs site and it's unclear. 

Up to you.

**Review by KevinMind — APPROVED** [review]
LGTM. Left a comment but not a blocker.

image

File: docs/topics/logs.md (hunk 1 above)
Comment (diox) [reply docs/topics/logs.md:9]: Yes, I tried that, that's actually how I originally had it, but it overflowed (the theme we're using only allows a `max-width` of `800px` on the content) so I figured I would make separate tables for now.

Comment (KevinMind) [reply docs/topics/logs.md:9]: Thought that might be the case 👍

**Pull request merged**